app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt

```diff
@@ -528,7 +528,7 @@ class RecentlyPlayedFragmentTest {
   }
 
   @Test
-  fun testRecentlyPlayedTestActivity_checkSpanForItem3_spanSizeIsOne() {
+  fun testRecentlyPlayedTestActivity_configChange_checkSpanForItem3_spanSizeIsOne() {
```

Comment:
For test at line 552
```diff
- testRecentlyPlayedTestActivity_configurationChange_checkSpanForItem1_spanSizeIsOne
+ testRecentlyPlayedTestActivity_configChange_checkSpanForItem1_spanSizeIsOne
```

Reply:
for others too, `configChange`. Please keep the consistency in all the names.

Reply:
There are two test with same name 
``testRecentlyPlayedTestActivity_configChange_checkSpanForItem1_spanSizeIsOne``
 but one is for landscape and other is of portrait

Reply:
So we have to modify the name according to that

Reply:
Should i change it to
``testRecentlyPlayedTestActivity_configChange_checkSpanForItem1_spanSizeIsOne`` to 
``testRecentlyPlayedTestActivity_configChangeLand_checkSpanForItem1_spanSizeIsOne``

and  ``configChangePot`` for portrait

Reply:
the test with name `testRecentlyPlayedTestActivity_configChange_checkSpanForItem1_spanSizeIsOne` is wrong as there is no configuration change happening in the test. 

update to 
testRecentlyPlayedTestActivity_checkSpanForItem1_spanSizeIsOne

Reply:
ok

Reply:
done